applications_tests/gls_navier_stokes_2d/gls_vof_hydrostat_mesh-adapt.output

```diff
@@ -28,36 +28,36 @@ Transient iteration : 3        Time : 0.06     Time step : 0.02     CFL : 2.1861
    Volume of triangulation:      1
    Number of VOF degrees of freedom: 553
 L2 error velocity : 3.31246e-06
-L2 error phase : 0.00656588
+L2 error phase : 0.00425561
 
 *****************************************************************
 Transient iteration : 4        Time : 0.08     Time step : 0.02     CFL : 1.96295e-05
 *****************************************************************
-   Number of active cells:       568
-   Number of degrees of freedom: 1950
+   Number of active cells:       463
+   Number of degrees of freedom: 1647
    Volume of triangulation:      1
-   Number of VOF degrees of freedom: 650
-L2 error velocity : 2.7637e-06
-L2 error phase : 0.00656595
+   Number of VOF degrees of freedom: 549
+L2 error velocity : 3.57164e-06
+L2 error phase : 0.00425569
 
 *****************************************************************
-Transient iteration : 5        Time : 0.1      Time step : 0.02     CFL : 1.89363e-05
+Transient iteration : 5        Time : 0.1      Time step : 0.02     CFL : 2.39966e-05
 *****************************************************************
-   Number of active cells:       784
-   Number of degrees of freedom: 2787
+   Number of active cells:       568
+   Number of degrees of freedom: 1950
    Volume of triangulation:      1
-   Number of VOF degrees of freedom: 929
-L2 error velocity : 3.56824e-06
-L2 error phase : 0.00656601
+   Number of VOF degrees of freedom: 650
+L2 error velocity : 2.76369e-06
+L2 error phase : 0.00425578
  time  error_velocity error_pressure 
-0.0200 3.071207e-05   0.0682         
-0.0400 4.143914e-06   0.0668         
-0.0600 3.312462e-06   0.0669         
-0.0800 2.763695e-06   0.0669         
-0.1000 3.568241e-06   0.0669         
+0.0200 3.071207e-05   0.9287         
+0.0400 4.143914e-06   0.9285         
+0.0600 3.312462e-06   0.9283         
+0.0800 3.571638e-06   0.9284         
+0.1000 2.763693e-06   0.9283         
```

Comment:
There is something wrong here. The pressure analytical solution has been redefined in the new version of Lethe so that it gives the right solution. i would watch that out here. You can compare with the regular hydrostat prm file for the right analytical solution.